src/components/Header/Header.js

```diff
@@ -111,7 +109,12 @@ class Header extends React.Component {
           }
 
           .logo {
-            border-radius: 65% 75%;
+            border-radius: 50%;
```

Comment:
Updating border-radius to 50% makes it so it looks like a circle rather than an oval.

![image](https://user-images.githubusercontent.com/14980353/47192147-32c48200-d300-11e8-927d-7fba5fe86012.png)